index.html

```diff
@@ -18,22 +19,38 @@
 
  <!-- navigation code here -->
 
+ <header class="main-header">
+
+  <h1>
+    SpaceWalkers
+  </h1>
+
+  <nav>
+    <a href="#">Home</a>
+    <a href="#">About Us</a>
+    <a href="#">Blog</a>
+    <a href="#">Contact</a>
+    <a href="#">Sign In</a>
+  </nav>
+</header>
+
   <div class="container home">
 
-    <header class="intro">
+    <section class="intro">
        <!-- image code here -->
-
+       <img src="/Users/mtrestman1/Lambda H&CSS/Sprint-Challenge--Advanced-CSS/img/main-header.png" class="earth" alt="Image of Earth">
```

Comment:
The way that you're accessing the image here will totally work on your computer, but if you want it to be visible on other computers, you'll need to access the image through the current directory (use the other img tags as examples)